projects/deja-js/component/tree-list/tree-list.component.ts

```diff
@@ -19,16 +22,30 @@ import { DejaTreeListScrollEvent } from './tree-list-scroll-event';
 @Component({
     changeDetection: ChangeDetectionStrategy.OnPush,
     encapsulation: ViewEncapsulation.None,
-    providers: [ViewPortService],
+    providers: [
+        ViewPortService,
+        { provide: MatFormFieldControl, useExisting: DejaTreeListComponent }
+    ],
     selector: 'deja-tree-list',
     styleUrls: [
         './tree-list.component.scss',
     ],
     templateUrl: './tree-list.component.html',
 })
-export class DejaTreeListComponent extends ItemListBase implements AfterViewInit, AfterContentInit, ControlValueAccessor {
-    /** Texte à afficher par default dans la zone de recherche */
-    @Input() public placeholder: string;
+export class DejaTreeListComponent extends ItemListBase implements CanUpdateErrorState, AfterViewInit, AfterContentInit, ControlValueAccessor, DoCheck, OnDestroy, MatFormFieldControl<any> {
+    public static nextId = 0;
+    @HostBinding() public id = `deja-tree-list-${DejaTreeListComponent.nextId++}`;
+    @HostBinding('class.floating') public get shouldLabelFloat() {
+        return this.focused || !this.empty;
+    }
+
+    @HostBinding('attr.aria-describedby') public describedBy = '';
+
+    public controlType = 'deja-list-tree';
```

Comment:
deja-tree-list

Reply:
Alors... sélection avec un model oui, t'as raison. Mais la fonctionnalité multi-select est déjà démontrée dans l'autre onglet de la demo exactement comme le composant select.

Donc pour moi c'est quelque chose comme "Minimal case" et "With models as an array of strings".